### PR TITLE
add initial user data source for polio

### PR DESCRIPTION
**Pull request by czue**
## Summary

https://dimagi-dev.atlassian.net/browse/SC-1291

This adds a static data source for the polio project so they can get metrics from the user document.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: custom/polio_rdc/ucr/data_sources/users.json [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    "config": {
+        "table_id": "commcare_users",
+        "display_name": "Users",
+        "referenced_doc_type": "CommCareUser",
+        "description": "Data from the User Model",
+        "base_item_expression": {},
+        "configured_filter": {},
+        "validations": [],
+        "configured_indicators": [
+            {
```
Comment (czue) [inline custom/polio_rdc/ucr/data_sources/users.json:17]: @idiene are you able to look through this and confirm it has everything you'd need? One obstacle is that we can't reproduce the `name` column without some dev, but I'm working on that.

File: custom/polio_rdc/ucr/data_sources/users.json [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "is_active",
+                "comment": null,
+                "create_index": false,
+                "datatype": "small_integer",
```
Comment (esoergel) [inline custom/polio_rdc/ucr/data_sources/users.json:148]: is there no bool type?

File: custom/polio_rdc/ucr/data_sources/users.json [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+                "type": "expression"
+            },
+            {
+                "column_id": "count",
+                "comment": null,
+                "create_index": false,
+                "datatype": "small_integer",
+                "display_name": "count",
+                "expression": {
+                    "constant": 1,
```
Comment (esoergel) [inline custom/polio_rdc/ucr/data_sources/users.json:279]: To be summed in the report config, I assume?

**Review by esoergel — APPROVED** [review]

File: custom/polio_rdc/ucr/data_sources/users.json (hunk 2 above)
Comment (czue) [reply custom/polio_rdc/ucr/data_sources/users.json:148]: it's quite odd. there is, but it throws a vaildation error if you try to use it. I didn't dig in further. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/datatypes.py#L9

File: custom/polio_rdc/ucr/data_sources/users.json (hunk 3 above)
Comment (czue) [reply custom/polio_rdc/ucr/data_sources/users.json:279]: yep

**Pull request merged**